api/task_lifecycle_client.go

```diff
@@ -1,14 +1,21 @@
 package api
 
 import (
+	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"net/http"
 
 	"github.com/hashicorp/consul-terraform-sync/api/oapigen"
 	"github.com/hashicorp/consul-terraform-sync/config"
 )
 
+const (
+	RunOptionInspect = "inspect"
+	RunOptionNow     = "now"
+)
+
```

Comment:
heh great minds think alike! Just a heads up that these constants were recently added to the [`api` package](https://github.com/hashicorp/consul-terraform-sync/blob/main/api/task.go#L22-L23).

It may be that it was recently merged to main, it's not included in your branch, but I suspect it'll cause a compile error if merged in. Could you rebase your branch?